src/Models/UserModel.php

```diff
@@ -320,4 +320,14 @@ protected function saveEmailIdentity(array $data): array
 
         return $data;
     }
+
+    /**
+     * Updates the user's last active date.
+     */
+    public function saveActiveDate(User $user): void
```

Comment:
`updateActiveDate()` is better?
Because it is just updating only.
*save* implies insert or update.

Reply:
I agree. It's done. I also added one more test that was missing for the `AccessTokens` authenticator

Reply:
Thank you!